contracts/wrapper/NameWrapper.sol

```diff
@@ -802,8 +796,18 @@ contract NameWrapper is
 
     /***** Internal functions */
 
-    function _canTransfer(uint32 fuses) internal pure override returns (bool) {
-        return fuses & CANNOT_TRANSFER == 0;
+    function _preTransferCheck(uint256 id, uint32 fuses, uint64 expiry) internal view override returns (bool) {
+        if(expiry < block.timestamp) {
+            // Tranferrible if the name is not emancipated
```

Comment:
typo on transferrible - also should we change it to name was* not emancipated. Since they are now expired and not emancipated.